frontend/src/components/pages/CampOverview/ViewCamperModal/PickupDropoffTableRow.tsx

```diff
@@ -54,7 +54,7 @@ const PickupDropoffTime = ({
   return (
     <Td pl={0} pt={2} pb={0} color={color}>
       <HStack>
-        {pickup ? <SunriseIcon fill={color} /> : <SunsetIcon fill={color} />}
+        {pickup ? <SunsetIcon fill={color} /> : <SunriseIcon fill={color} />}
```

Comment:
I believe need to revert this - pickup should be the sunrise button (I know it's kinda confusing)

Expected look: 
![Screen Shot 2023-01-28 at 2 12 05 PM](https://user-images.githubusercontent.com/46568041/215287338-c0c5b1f7-0ebd-4b7c-b8df-7a240e0b8c82.png)

Design ref:
![Screen Shot 2023-01-28 at 2 13 30 PM](https://user-images.githubusercontent.com/46568041/215287348-f1bbcf73-67f0-49a5-bef7-483b8b5f3403.png)

Reply:
Sounds good, I'll revert it but I think I'll make a lower priority ticket just going over the inconsistencies of these icons. 
![image](https://user-images.githubusercontent.com/30396273/215349168-ec9f90a7-ff61-40e0-b6c3-73e836c3658a.png)
Here, early dropoff is sunrise and late pickup is sunset (which IMO, makes a lot more sense, but we can verify with the designers)

Reply:
yep sounds good, i totally thought it was the other way around initially as well but I feel like if we are gonna make a change we should choose the easier one to change (ie the most consistent set we have rn)